frontend/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -512,6 +519,7 @@ const DashboardPage = ({ router, location }: IDashboardProps): JSX.Element => {
       } finally {
         setUpdatingActivityFeedAutomations(false);
         refetchConfig();
+        refetchActivities();
```

Comment:
refetchActivitiesRef.current && refetchActivitiesRef.current(); ?
idk haven't tried it

Reply:
Thanks for going over this in frontend sync! Def need to follow a similar pattern on HDP. Just realized host activity refetches on any close of run script modal >.<